30 paskaita (egzaminas)/Task 7/script.js

```diff
@@ -10,3 +10,10 @@ const audi = {
   year: 2005,
   color: 'white',
 };
+
+function showObjectKeys () {
+  const newArray = Object.keys(audi)
```

Comment:
Cia pratima supratau pazodziui, del to ir i array sudejay key pavadinimus, o ne reiksmes.
Jei reikia reiksmiu tai rasyciau aisku const newArray = Object.values(audi)

Reply:
Tteisingai, taip ir reikejo 🙂